src/commercial.consentless.ts

```diff
@@ -23,15 +20,6 @@ const bootConsentless = async (
 
 	//this is added so that we can load the subscriber cookie for DCR pages and correctly hide ads
```

Comment:
Can this comment be deleted too?

Reply:
These were remnant comments from the original user-features and though it would be useful to keep track to explain the various functions?